### ISSUE TITLE
No gap between icon in visits in real time widget

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#21310]
This is using Matomo 5.

When viewing the real time widget in a dashboard, then there is no gap between the Google icon and the word "Google".

<img width="258" alt="image" src="https://github.com/matomo-org/matomo/assets/273120/48b59efd-5932-4596-a348-aa3fc19f5883">

After a refresh of the widget (or when the widget auto updates), the gap seems to be there.

Browser: Chrome, latest version

**Comment by michalkleiner** [comment]
So is this on the first load of the dashboard and then it disappears? Is it replicable, does it happen every time even after a hard refresh and/or in an incognito window with caches disabled?

**Comment by tsteur** [comment]
It happens for me every time on Matomo 5.

It's not related to caches.

**Comment by michalkleiner** [comment]
I can confirm this behaviour on Sandbox. There's no gap on first dashboard load, then subsequent update of the live visits widget looks normal. Tested in Brave. Haven't seen any obvious markup or styling differences at a glance.

**Comment by sgiehl** [comment]
Might be a regression of the vue migration. Looking at the code it seems that the data from the initial request is inserted after removing any whitespaces between the html tags. Responses from further requests are inserted with whitespace, which seems to be the cause, that on the first request there is no whitespace, while further requests add one...

**Comment by michalkleiner** [comment]
Yeah, that might be the case because in elements view in devtools the markup looks the same but it's the browser formatting and it may be obscuring there's no whitespace between the tags, so the raw request response markup needs to be examined. Should be an easy fix adding a non-breakable space there.
